frontend/file/build.gradle.kts

```diff
@@ -42,6 +43,8 @@ sourceSets["main"].java { srcDir(additionalSourceDirs) }
 
 val generateMacheteParser =
   tasks.withType<GenerateParserTask> {
+    // Just to make sure correct version of the task is used
+    javaLauncher.set(javaLauncher.get())
```

Comment:
@MaciejG604 what specifically happens without this hack?

Reply:
A compilation error would happen if the plugin was not correctly loaded from mavenLocal. Just a check for myself.

Reply:
Aaa I see now, makes sense